queries/wof.jinja2

```diff
@@ -12,7 +12,7 @@ SELECT
       'area', area::bigint,
       'min_zoom', min_zoom,
       'max_zoom', max_zoom,
-      'is_landuse_aoi', CASE WHEN is_landuse_aoi = TRUE THEN TRUE END,
+      'is_landuse_aoi', is_landuse_aoi,
```

Comment:
We have an hstore column for the l10n names, which is what we stick into `__properties__` at the moment. We could also merge these values from `__places_properties__` into `__properties__`, but  we will still need an empty `__places_properties__` jsonb value to signal that the record should be included in the places layer. That would look something like:

```
COALESCE(to_jsonb(l10n_name), '{}'::jsonb) ||
jsonb_build_object(
  'name', name,
  'source', 'whosonfirst.mapzen.com',
  'mz_n_photos', n_photos,
  'area', area,
  'min_zoom', min_zoom,
  'max_zoom', max_zoom,
  'is_landuse_aoi', is_landuse_aoi,
  'placetype', wof_np.placetype_string
  ) AS __properties__,

'{}::jsonb' AS __places_properties__
```

Is that better?